native/src/lib.rs

```diff
@@ -632,8 +632,8 @@ struct CreateProofContext {
 /// {
 ///     "proof": ArrayBuffer,                   // The proof from `bbs_create_proof`
 ///     "publicKey": ArrayBuffer,               // The public key of the signer in BLS form
-///     "messages": [String, String]            // The revealed messages as strings. They will be Blake2b hashed.
-///     "nonce": String                         // This is an optional nonce from the verifier and will be used in the proof of committed messages if present. It is strongly recommend that this be used.
+///     "messages": [ArrayBuffer, ArrayBuffer]  // The revealed messages as strings. They will be Blake2b hashed.
+///     "nonce": ArrayBuffer                    // This is an optional nonce from the verifier and will be used in the proof of committed messages if present. It is strongly recommend that this be used.
```

Comment:
similar to above about details of consequences if this isn't used.

Reply:
Additionally are there any constraints about size or generation like must be generated with RNG, must only be used once, etc?

Reply:
The recommended size is 12 bytes &le; x &le; 32 bytes. More than 32 bytes is just overkill.

Reply:
Should we add this detail into the API comment?